docs/cri/config.md

```diff
@@ -248,6 +248,7 @@ version = 3
 
     [plugins.'io.containerd.cri.v1.runtime'.cni]
       bin_dir = '/opt/cni/bin'
+      bin_dirs = []
```

Comment:
should probably update this part of the description.. comment out the bin dir as deprecated.. show the array with the default dir..